osu.Game/Overlays/ChatOverlay.cs

```diff
@@ -478,11 +478,35 @@ private void postMessage(TextBox textbox, bool newText)
                 return;
             }
 
+            bool isAction = false;
+
             if (postText[0] == '/')
             {
-                // TODO: handle commands
-                target.AddNewMessages(new ErrorMessage("Chat commands are not supported yet!"));
-                return;
+                postText = postText.Substring(1);
+                string commandKeyword = postText.Split(' ')[0];
+
+                switch (commandKeyword)
+                {
+                    case "me":
+
+                        if (!postText.StartsWith("me ") || string.IsNullOrWhiteSpace(postText.Substring(3)))
+                        {
+                            currentChannel.AddNewMessages(new ErrorMessage("Usage: /me [action]"));
+                            return;
+                        }
+
+                        isAction = true;
+                        postText = postText.Substring(3);
+                        break;
+
+                    case "help":
+                        currentChannel.AddNewMessages(new ErrorMessage("Supported commands: /help, /me [action]"));

```

Comment:
not sure this really counts as an `ErrorMessage`

Reply:
Should those info messages be displayed something like this: https://puu.sh/xG6CE/36e41bda09.png ?